### PR TITLE
Update python packages.csv for newly onboarded track 2 packages

### PR DESCRIPTION
**Pull request by nickzhums**

### CODE REVIEW

File: _data/releases/latest/python-packages.csv [hunk 1]
```diff
@@ -86,7 +86,7 @@
 "azure-mgmt-avs","0.1.0","","azure-mgmt-avs","","NA","NA","","","",""
 "azure-mgmt-azurestack","0.1.0","","azure-mgmt-azurestack","Azurestack","azurestack","NA","","","",""
 "azure-mgmt-azurestackhci","","1.0.0rc1","azure-mgmt-azurestackhci","","NA","NA","NA","","",""
-"azure-mgmt-baremetalinfrastructure","","1.0.0b1","azure-mgmt-baremetalinfrastructure","","NA","NA","NA","","",""
+"azure-mgmt-baremetalinfrastructure","","1.0.0b1","Resource Management - Bare Metal Infrastructure","Bare Metal","","","","mgmt","",""
```
Comment (weshaggard) [inline _data/releases/latest/python-packages.csv:89]: Looks like some docs links are broken so you can put and NA in those columns to fix those until they are published, at which time our automation will update this table.

Comment (nickzhums) [reply _data/releases/latest/python-packages.csv:89]: got it, will do

**Review by weshaggard — APPROVED** [review]

File: _data/releases/latest/python-packages.csv [hunk 2]
```diff
@@ -95,15 +95,15 @@
...
 "azure-mgmt-containerregistry","2.8.0","","azure-mgmt-containerregistry","Container Registry","https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-containerregistry_2.8.0/azure-mgmt-containerregistry/","https://docs.microsoft.com/python/api/overview/azure/container-registry?view=azure-python","NA","","",""
 "azure-mgmt-containerservice","9.4.0","","azure-mgmt-containerservice","Container Service","containerservice","NA","","","",""
 "azure-mgmt-core","1.2.0","","azure-mgmt-core","Core","core","NA","","","",""
 "azure-mgmt-cosmosdb","1.0.0","","azure-mgmt-cosmosdb","Cosmos DB","cosmos","https://docs.microsoft.com/python/api/overview/azure/cosmosdb?view=azure-python","","","",""
 "azure-mgmt-costmanagement","0.2.0","","azure-mgmt-costmanagement","Costmanagement","costmanagement","NA","","","",""
-"azure-mgmt-customproviders","0.1.0","","azure-mgmt-customproviders","Custom Providers","customproviders","NA","","","",""
+"azure-mgmt-customproviders","0.1.0","","azure-mgmt-customproviders","Custom Providers","customprCoviders","NA","","","",""
```
Comment (nickzhums) [inline _data/releases/latest/python-packages.csv:106]: @weshaggard is customprCoviders a typo?

Comment (weshaggard) [reply _data/releases/latest/python-packages.csv:106]: Yes 👍

**Review by weshaggard — APPROVED** [review]

**Pull request merged**